### DISCUSSION TITLE
Custom ContentDialog doens't get the focus on a Skia project

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#9545] [General]
Hi, I'm using a custom ContentDialog to get a string from a USB RFID reader. The reader acts as a keyboard and I use the KeyUp event handler to get the value:

`KeyUp += RfidDialog_KeyUp;` 

It works fine in WinUI but in the Skia project focus seems to be hooked in the background frame. You can see the white rectangle focusing the settings button.

[Focus rectangle](https://i.imgur.com/qshqdFO.png)

If I press Tab key, focus alternates between Settings and User button but the ContentDialog never receives the focus so it never gets KeyUp events.

Besides CornerRadius is displayed differently in both projects.

Here a link with both images:

[CornerRadius](https://imgur.com/a/ysKSgYD)

**Comment by MartinZikmund** [comment]
Definitely a problem caused by our Popup implementation which does not yet play nice with focus.  Will take care of fixing it 👍

**Reply by mcNets** [reply]
Hi @MartinZikmund, any news about it?

**Reply by MartinZikmund** [reply]
Hi @mcNets sorry for the delay. Can you try the latest dev bits for the corner radius issue? I think that should be resolved now. I am working on the focus handling now

**Reply by mcNets** [reply]
@MartinZikmund  Sure, I'll keep you posted.

**Reply by mcNets** [reply]
@MartinZikmund I've used `<PackageReference Include="Uno.WinUI.Skia.Gtk" Version="4.6.0-dev.984" />` and still the same aspect.

**Reply by mcNets** [reply]
@MartinZikmund it's fine now. Thanks

**Reply by MartinZikmund** [reply]
Good to know! The focus handling is working correctly too?

**Reply by mcNets** [reply]
Yes, and App.Exit() throws no exception now. 👍
